packages/plugin-react-swc/playground/emotion/src/index.tsx

```diff
@@ -1,6 +1,6 @@
+import { App } from './App.tsx'
```

Comment:
It feels weird for me

Reply:
Yeah, we need to wait for proper import sorting. It looks like it's only sorting by string, so relative import comes first `. < @ < a`.